LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2022-2024 ALEPH.IM SAS
```

Comment:
Isn't it a SASU?

Reply:
SAS according to the French government: https://annuaire-entreprises.data.gouv.fr/entreprise/aleph-im-882412869

> Sa forme juridique est SAS, société par actions simplifiée.

(SASU is a special form of SAS, so it may be both, but apparently the official term is SAS).